<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AiSetup</class>
 <widget class="QWidget" name="AiSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*局部样式表*/
QLabel#img_Lab{
	border: 1px solid black;
	border-radius:4px;
}

QLineEdit#Ainame_Edit{
	border: 1px solid black;
	color:#4A7D7D;
	font:normal 20px 黑体;
	border-radius:4px;
	background-color: rgba(250, 250, 250,0.8);
}

QLineEdit#Ainame_Edit:hover{
	background-color: rgb(255, 255, 255);
}

QLineEdit#Aisign_Edit{
	border: 1px solid black;
	color:#4A7D7D;
	font:normal 20px 黑体;
	border-radius:4px;
	background-color: rgba(250, 250, 250,0.8);
}

QLineEdit#Aisign_Edit:hover{
	background-color: rgb(255, 255, 255);
}

QGroupBox#getType_Box{
	border: 1px solid red;
}

QGroupBox#APIKey_Box{
	border: 1px solid red;
}

QGroupBox#APISecret_Box{
	border: 1px solid red;
}

QGroupBox#getPath_Box{
	border: 1px solid red;
}

QLineEdit#APIKey_Edit{
	color:#4A7D7D;
	font:normal 20px 黑体;
	border-radius:4px;
	background-color: rgba(250, 250, 250,0.8);
}

QLineEdit#APIKey_Edit:hover{
	background-color: rgb(255, 255, 255);
}

QLineEdit#APISecret_Edit{
	color:#4A7D7D;
	font:normal 20px 黑体;
	border-radius:4px;
	background-color: rgba(250, 250, 250,0.8);
}

QLineEdit#APISecret_Edit:hover{
	background-color: rgb(255, 255, 255);
}

QPushButton#determine_Btn{
	border: 1px solid black;
	border-radius: 5px;
	background-color: rgba(250, 250, 250,0.8);
	font:normal 18px &quot;楷体&quot;;
	color: rgb(0, 0, 0);
}

QPushButton#determine_Btn:hover{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QLabel" name="img_Lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../image/Image.qrc">:/new/user/user/Ai_Logn.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Ainame_Edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>机器人名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Aisign_Edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>360</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>360</width>
     <height>30</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>机器人介绍</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="APIKey_Box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>470</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>470</width>
     <height>60</height>
    </size>
   </property>
   <property name="title">
    <string>APIKey</string>
   </property>
   <widget class="QLineEdit" name="APIKey_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>450</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>PbhRuT6iTPgpvETgFngOWAGa</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="APISecret_Box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>470</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>470</width>
     <height>60</height>
    </size>
   </property>
   <property name="title">
    <string>APISecret</string>
   </property>
   <widget class="QLineEdit" name="APISecret_Edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>450</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>fKey53SS5G7d56EPH3LTeMINtKpAYI9V</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="getPath_Box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>470</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>470</width>
     <height>60</height>
    </size>
   </property>
   <property name="title">
    <string>请求地址</string>
   </property>
   <widget class="QLabel" name="getPath_Lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>450</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>450</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>https://aip.baidubce.com/oauth/2.0/token</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="getType_Box">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>80</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>60</height>
    </size>
   </property>
   <property name="title">
    <string>请求类型</string>
   </property>
   <widget class="QLabel" name="getTypr_Lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POST</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="determine_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>470</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>470</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>470</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../image/Image.qrc"/>
 </resources>
 <connections/>
</ui>
